app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -257,11 +267,10 @@ export default function createRPCMethodTrackingMiddleware({
             req.securityAlertResponse.description;
         }
         ///: END:ONLY_INCLUDE_IF
-
         const isConfirmationRedesign =
           isConfirmationRedesignEnabled() &&
-          [...REDESIGN_APPROVAL_TYPES, ...REDESIGN_TRANSACTION_TYPES].find(
-            (type) => type === method,
```

Comment:
I removed the redesign transaction types check because it does not work. We will handle this with https://github.com/MetaMask/metamask-extension/issues/24486